### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by jdbranham**

### CODE REVIEW

**Comment by jdbranham** [comment]
If we rebuild without cache, I think it will succeed.

File: .jscs.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
-    "esnext": true,
```
Comment (jdbranham) [inline .jscs.json]: Default behavior now. No longer required

File: Gruntfile.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ module.exports = function(grunt) {
 
   require('load-grunt-tasks')(grunt);
 
-  grunt.loadNpmTasks('grunt-execute');
```
Comment (jdbranham) [inline Gruntfile.js]: Not required AFAIK

File: package.json [hunk 3]
```diff
@@ -16,30 +16,28 @@
     "url": "https://github.com/raintank/app-plugin-worldping/issues"
   },
   "devDependencies": {
-    "babel": "~6.5.1",
-    "grunt": "~0.4.5",
+    "babel-cli": "^6.24.1",
```
Comment (jdbranham) [inline package.json]: Moved dependencies to "devDependency" section, as they are not required to run the plugin once it's built.

**Comment by DanCech** [comment]
Rebuild with clean cache says: cannot find module load-grunt-tasks

**Comment by jdbranham** [comment]
Thanks @DanCech 
I have some deps installed globally so didn't notice that one missing.

**Comment by DanCech** [comment]
dependency stuff looks good, but this PR also has the snapshot updates in it.  Can you keep those in their own branch?

**Comment by DanCech** [comment]
Still showing the snapshot stuff, let's just get that merged first and come back to this.

**Comment by jdbranham** [comment]
Yep. Looks like I didn't do that right. =\

**Comment by jdbranham** [comment]
Figured out I had previously rebased against my local master which had the snapshot changes. 
Should be cool now!

**Pull request merged**